requirements.txt

```diff
@@ -1,35 +1,26 @@
 amqp==5.1.1
-asgiref==3.5.2
-async-timeout==4.0.2
-pyjwt
-flask_mail
-flask_jwt_extended
+attrs==21.2.0
 bcrypt==3.2.0
-billiard==3.6.4.0
-blinker==1.4
 celery==5.2.7
-certifi==2020.6.20
-chardet==4.0.0
-click==8.0.3
-click-didyoumean==0.3.0
-click-plugins==1.1.1
-click-repl==0.2.0
-cryptography==3.4.8
-flask
-Flask
-Flask-SQLAlchemy
+fasteners==0.14.1
+Flask==2.2.2
+Flask-Mail==0.9.1
+Flask-SQLAlchemy==3.0.2
 Flask-WTF==1.0.1
-future==0.18.2
-greenlet==2.0.1
 Jinja2==3.1.2
-Mako==1.1.3
+jsonschema==3.2.0
```

Comment:
Please make sure to include those requirements which are used in the project.

Reply:
Some how my pip3 freeze command includes unnecessary requirements working on it.